frontend/frontend.containerfile

```diff
@@ -1,15 +1,19 @@
-FROM node:14.15.3-alpine3.12 AS build
+# pull official base image
```

Comment:
The nodejs server is only for development. This image build code should be in frontend-dev.

Reply:
we shouldn't need to use nginx because proxying is no longer needed. Realistically we don't even need the two different dockerfiles